Dockerfile

```diff
@@ -1,9 +1,9 @@
-FROM rustlang/rust@sha256:5af55c68b21232886d8d9bd35563b8a2ac0f71952369fb71346a51b331acd0d4 as rust_builder
+FROM rustlang/rust@sha256:ff3e6405319286ee5ff427ffdda577760507774f4c1dd7bab12ebc54ad31a696 as rust_builder
 RUN rustup target add x86_64-unknown-linux-musl
 RUN apt-get update && apt-get install -y musl-tools
+ARG CACHE_DATE=2016-01-01
 RUN git clone https://github.com/brave-intl/challenge-bypass-ristretto-ffi /src
 WORKDIR /src
-RUN git checkout 1.0.0-pre.1
```

Comment:
ah shoot, sorry can we push a new tag for 1.0.0-pre.4 and restore this?

Reply:
done